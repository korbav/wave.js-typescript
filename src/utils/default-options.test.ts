import defaultOptions from './default-options';

describe('defaultOptions', () => {
  it('should match the expected set of options', () => {
    expect(defaultOptions).toMatchSnapshot();
  });
});
