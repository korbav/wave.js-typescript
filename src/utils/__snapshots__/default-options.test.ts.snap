// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultOptions should match the expected set of options 1`] = `
Object {
  "colors": Array [
    "#d92027",
    "#ff9234",
    "#ffcd3c",
    "#35d0ba",
  ],
  "getGlobal": [Function],
  "globalAccessKey": "$wave",
  "setGlobal": [Function],
  "stroke": 1,
  "type": "bars",
}
`;
